src/CLRunner.ts

```diff
@@ -584,14 +582,14 @@ export class CLRunner {
 
             // Handle any other non-message input, filter out empty messages
             if (activity.type !== BB.ActivityTypes.Message || !activity.text || activity.text === "") {
-                await InputQueue.MessageHandled(state.MessageState, activity.id)
+                await InputQueue.MessageHandled(state.MessageState, activity.conversation.id, activity)
```

Comment:
Feels a little weird to pass the conversation id separatly from the activity since they could now be incorrect rather than getting the id from the activity but I don't feel strongly

Reply:
Agreed,  Had to do it this way, because in some cases the activity is undefined, but the conversationId is still needed